client/package.json

```diff
@@ -8,6 +8,7 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "prettier": "prettier --ignore-path .gitignore --write src",
     "preview": "vite preview"
   },
   "dependencies": {
```

Comment:
Side todo or maybe you can do it, but "@type" dependencies should be in `devDependencies`. See the following: [Item 45: Put TypeScript and @types in devDependencies](https://www.oreilly.com/library/view/effective-typescript/9781492053736/ch06.html) for further explanation